cmd/stack_package.go

```diff
@@ -75,13 +75,15 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var stackPackageCmd = &cobra.Command{
 		Use:   "package",
-		Short: "Package a stack in the local Appsody environment",
-		Long: `This command is a tool for stack developers to package a stack from their local Appsody development environment. Once the stack is packaged it can then be tested via Appsody commands. The package command performs the following:
-- Creates/updates an index file named "dev.local-index.yaml" and stores it in .appsody/stacks/dev.local
-- Creates a tar.gz for each stack template and stores it in .appsody/stacks/dev.local
-- Builds a Docker image named "dev.local/[stack name]:SNAPSHOT"
-- Creates an Appsody repository named "dev.local"
-- Adds/updates the "dev.local" repository of your Appsody configuration`,
+		Short: "Package your stack.",
+		Long: `Package your stack in a local Appsody development environment. You must run this command from the root directory of your stack.
+
+The packaging process builds the stack image, generates the "tar.gz" archive files for each template, and adds your stack to the "dev.local" repository in your Appsody configuration. You can see the list of your packaged stacks by running "appsody list dev.local".`,
```

Comment:
"appsody list dev.local" looks strange in double quotes. Can't we use single quotes '' (I'm sure we can't use backticks)?

Reply:
I think they all have here. Am I missing something?